apps/system/utils/Kconfig

```diff
@@ -62,6 +62,13 @@ config ENABLE_CPULOAD
 		CPU load monitor is a daemon that will periodically assess
 		CPU usage by all live tasks and threads in the system.
 
+config PM_UTIL
+	bool "Direct PM state transit utility"
+	default n
+	depends on PM && ARCH_SUPPORT_PM_UTIL
```

Comment:
1. PM_UTIL only calls pm_sleep function which defines with ARCH_SUPPORT_PM_UTIL.
2. ARCH_SUPPORT_PM_UTIL already has a dependency with PM.
so that I think we don't need pm here.

Reply:
Agree with you.
pm util is not needed now, so I have removed it in later commits.